pkg/controller/vpc_nat_gateway.go

```diff
@@ -33,14 +33,14 @@ var (
 )
 
 const (
-	NAT_GW_INIT             = "init"
-	NAT_GW_FLOATING_IP_SYNC = "floating-ip-sync"
-	NAT_GW_EIP_ADD          = "eip-add"
-	NAT_GW_EIP_DEL          = "eip-del"
-	NAT_GW_SNAT_SYNC        = "snat-sync"
-	NAT_GW_DNAT_SYNC        = "dnat-sync"
-	NAT_GW_SUBNET_ROUTE_ADD = "subnet-route-add"
-	NAT_GW_SUBNET_ROUTE_DEL = "subnet-route-del"
+	NatGwInit           = "init"
+	NatGwFloatingIpSync = "floating-ip-sync"
+	NatGwEipAdd         = "eip-add"
+	NatGwEipDel         = "eip-del"
+	NatGwSnatSync       = "snat-sync"
+	NatGwDnatSync       = "dnat-sync"
+	NatGwSubnetRouteAdd = "subnet-route-add"
+	NatGwSubnetRouteDel = "subnet-route-del"
```

Comment:
We should also make the first characters lowercase.

Reply:
The latest code has been commited. First characters have been changed lowercase.

Reply:
This commit have been confused. I will make a new commit.